metis-validation/metis-validation-rest/src/main/java/eu/europeana/validation/rest/ValidationController.java

```diff
@@ -87,8 +87,8 @@ public ValidationResult validate(
   }
 
   /**
```

Comment:
Fix code smell in line 74 for variable ```record``` a rename could be ```recordToValidate```
also remember to update the javadoc.

```
 @PostMapping(value = RestEndpoints.SCHEMA_VALIDATE, consumes = APPLICATION_XML_VALUE, produces = APPLICATION_JSON_VALUE)
  @ResponseBody
  @ApiOperation(value = "Validate single record based on schema", response = ValidationResult.class)
  public ValidationResult validate(
      @ApiParam(value = "schema") @PathVariable("schema") String targetSchema,
      @RequestBody String recordToValidate
  ) throws ValidationException {
    if (!schemaProvider.isPredefined(targetSchema)) {
      throw new ValidationException("", "", "It is not predefined schema.");
    }

    ValidationResult result = validator.singleValidation(targetSchema, null, null, recordToValidate);
    if (result.isSuccess()) {
      return result;
    } else {
      LOGGER.error(result.getMessage());
      throw new ValidationException(result.getRecordId(), result.getNodeId(), result.getMessage());
    }
  }
```